code/game/objects/objs.dm

```diff
@@ -1,36 +1,48 @@
 /obj
-	//var/datum/module/mod		//not used
-	var/origin_tech = null	//Used by R&D to determine what research bonuses it grants.
 	animate_movement = SLIDE_STEPS
-	var/list/species_exception = null	// list() of species types, if a species cannot put items in a certain slot, but species type is in list, it will be able to wear that item
-	var/sharp = FALSE		// whether this object cuts
-	var/in_use = FALSE // If we have a user using us, this will be set on. We will check if the user has stopped using us, and thus stop updating and LAGGING EVERYTHING!
+	/// Used by R&D to determine what research bonuses it grants.
+	var/origin_tech = null
+	/// `list()` that allows any contained species to wear the item.
+	var/list/species_exception = null
+	/// Can this object cut?
+	var/sharp = FALSE
+	/// If we have a user using us, this will become `TRUE`. We will check if the user has stopped using us, and thus stop updating and LAGGING EVERYTHING!
+	var/in_use = FALSE
+	/// What type of damage does this object deal?
 	var/damtype = "brute"
+	/// How much damage this object does in melee.
 	var/force = 0
 	// You can define armor as a list in datum definition (e.g. `armor = list("fire" = 80, "brute" = 10)`),
 	// which would be converted to armor datum during initialization.
 	// Setting `armor` to a list on an *existing* object would inevitably runtime. Use `getArmor()` instead.
 	var/datum/armor/armor
-	var/obj_integrity	//defaults to max_integrity
+	/// Health of the object. If unspecified, defaults to `max_integrity`.
+	var/obj_integrity
+	/// Maximum health of the object, and default value of `obj_integrity`.
 	var/max_integrity = 500
-	var/integrity_failure = 0 //0 if we have no special broken behavior
-	///Damage under this value will be completely ignored
+	/// Health threshold below which the object will break. Defaults to 0 for no special broken behavior.
+	var/integrity_failure = 0
+	/// Damage under this value will be completely ignored.
 	var/damage_deflection = 0
-
-	var/resistance_flags = NONE // INDESTRUCTIBLE
-	var/custom_fire_overlay // Update_fire_overlay will check if a different icon state should be used
-
-	var/acid_level = 0 //how much acid is on that obj
-
-	var/can_be_hit = TRUE //can this be bludgeoned by items?
-
+	/// Flags that make this object harder to destroy, e.g. ACID_PROOF, FIRE_PROOF, INDESTRUCTIBLE.
```

Comment:
```suggestion
	/// Flags that make this object harder to destroy, e.g. [ACID_PROOF], [FIRE_PROOF], [INDESTRUCTIBLE].
```
dmdoc will link these if those defines have documentation. they do but they're not autodocced but that can be fixed later